R/get_gene_expression.R

```diff
@@ -5,11 +5,16 @@
 #' @details Effectively get gene expression for one or multiple genes for al GAMBL samples.
 #' This function can also take an already loaded expression matrix (`expression_data`)
 #' to prevent the user from having to load the full expression matrix if this function needs to be run in an interactive session.
-#' For examples and more info, refer to the parameter descriptions as wella s vignette examples.
+#' For examples and more info, refer to the parameter descriptions as well as vignette examples.
+#' The function has argument `engine`, which accepts string "read_tsv", "grep", "vroom", and "fread". This will determine the way
+#' the data is imported into R. When testing on GSC, the grep was the fastest but with a lot of variation in the run time (anywhere between 4-10 min).
+#' Other engines produced similar run times (~ 7 min) on GSC, with vroom engine being the most consistent one. However, on other
+#' systems (especially with fast hard drives and MacBooks for remote users) the read_tsv engine was significantly faster than the grep.
 #'
 #' @param metadata GAMBL metadata.
```

Comment:
Perhaps, this is a good time to harmonize this parameter name (`these_samples_metadata`)? In addition, I think the parameter description could be a bit more informative.

Reply:
Thanks,  have added this :+1: